integration-cli/docker_cli_help_test.go

```diff
@@ -100,5 +87,9 @@ func TestCmdHelpWidth(t *testing.T) {
 		}
 	}
 
+	if count < 10 {
```

Comment:
I dunno, you can see how many commands now. New commands are very rare in docker, so you can fix this test every time you add one.